test/docker-compose.yml

```
@@ -104,6 +105,38 @@ services:
       test_net:
         ipv4_address: 172.16.238.50
 
+  lumberjack:
+    image: uavaustin/lumberjack
+    ports:
+      - '6000:6000'
+    depends_on:
+      - 'forward-interop'
+      - 'pong'
+    networks:
+      test_net:
+        ipv4_address: 172.16.238.17
+
+  influx:
+    image: influxdb:alpine
+    ports:
+      - '8086:8086'
+    volumes:
+      - './influxdb/data:/var/lib/influxdb'
+      - './influxdb/config/:/etc/influxdb/'
+    networks:
+      test_net:
+        ipv4_address: 172.16.238.18
+
+  grafana:
+    image: grafana/grafana
+    ports:
+      - '3000:3000'
```

Also depends on influx. It will not show anything meaningful, but at least the primary data connection will work.
```suggestion
      - '3000:3000'
    depends_on:
      - 'influx'
```